src/plays/savings-calculator/components/SavingOptions/index.jsx

```diff
@@ -0,0 +1,150 @@
+          id="currency"
+          value={currency}
+          onChange={(e) => setCurrency(e.target.value)}
+        >
+          <option>INR</option>
+          <option>USD</option>
+        </select>
+      </div>
+      <form>
+        <div className="savingOptions__formGroup">
+          <label htmlFor="starting-balance">Starting Balance:</label>
+          <div className="savingOptions__inputGroup">
+            <div className="savingOptions__inputPrepend">
+              {currency === "INR" ? "₹" : "$"}
+            </div>
+            <input
+              type="number"
+              className="savingOptions__formControl"
+              id="starting-balance"
+              placeholder="Enter initial balance"
+              value={startingBalance}
+              onChange={(e) => setStartingBalance(e.target.value)}
+            />
+          </div>
+        </div>
+        <div className="savingOptions__formGroup">
+          <label htmlFor="monthly-contribution">Monthly Contribution:</label>
+          <div className="savingOptions__inputGroup">
+            <div className="savingOptions__inputPrepend">
+              {currency === "INR" ? "₹" : "$"}
+            </div>
+            <input
+              type="number"
+              className="savingOptions__formControl"
+              id="monthly-contribution"
+              placeholder="Enter monthly contribution"
+              value={monthlyContribution}
+              onChange={(e) => setMonthlyContribution(e.target.value)}
+            />
+          </div>
+        </div>
+        <div className="savingOptions__formGroup">
+          <label htmlFor="period">Period:</label>
+          <div className="savingOptions__inputGroup">
+            <input
+              type="number"
+              className="savingOptions__formControl"
+              id="period"
+              placeholder="Enter period of contribution"
+              value={period}
+              onChange={(e) => setPeriod(e.target.value)}
+            />
+          </div>
+        </div>
+        <div className="savingOptions__periodChoice">
+          <div className="savingOptions__formRadio">
+            <input
+              className="savingOptions__radioInput"
+              type="radio"
+              name="period-choice"
+              id="period-choice-years"
+              value="years"
+              defaultChecked
+              onChange={() => setPeriodChoice("Years")}
+            />
+            <label
+              className="savingOptions__radioLabel"
+              htmlFor="period-choice-years"
+            >
+              Years
+            </label>
+          </div>
+          <div className="savingOptions__formRadio">
+            <input
+              className="savingOptions__radioInput"
+              type="radio"
+              name="period-choice"
+              id="period-choice-months"
+              value="months"
+              onChange={() => setPeriodChoice("Months")}
```

Comment:
@kd100100 , What we are trying to do here?
Why we are directly setting the `Months` as value and not taking the value from the onChange event?

Reply:
Modified onChange